includes/discovery/storage/ericsson-ipos.inc.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/*
+ * Copyright (c) 2024 Rudy Broersma <tozz@kijkt.tv>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * eriRouterSRStorageSize is reported in KBytes
+ */
+
+if ($device['os'] == 'ericsson-ipos') {
+    $eri_filesystem = snmpwalk_cache_oid($device, 'eriRouterSRStorageTable', [], 'ERICSSON-ROUTER-SYS-RESOURCES-MIB');
```

Comment:
Could you replaces calls to snmpwalk_cache_oid with the newer `SnmpQuery::walk` form ? 
Something like this :`$eri_filesystem = SnmpQuery::hideMib()->walk('ERICSSON-ROUTER-SYS-RESOURCES-MIB::eriRouterSRStorageTable)->table();`
_(not tested)_